### ISSUE TITLE
Collapse navigation doesn't work -- no [+] displayed

### ISSUE THREAD

**Issue by yannikschaelte** [readthedocs/sphinx_rtd_theme#1462]
### Problem

Although setting

```python
html_theme_options = {
    'collapse_navigation': False,
    #'navigation_depth': -1,
}
```

in `conf.py` (see [collapse_navigation](https://sphinx-rtd-theme.readthedocs.io/en/stable/configuring.html)), no [+] links are displayed. This behavior changed in version 1.2.0:

![image](https://user-images.githubusercontent.com/31767307/234848399-3e446773-d791-4a3e-a185-81ad268299e6.png)

Compared to 1.1.0:

![image](https://user-images.githubusercontent.com/31767307/234847978-9d0de0a0-99e1-4c14-9e83-6a9219f97208.png)

#### Expected Results

Expanding/collapsing the navigation to the specified level should be possible.

### Environment Info

- Python Version: 3.10
- Sphinx Version: 6.2.1
- RTD Theme Version: 1.2.0

**Comment by benjaoming** [comment]
Hi @yannikschaelte 

Thanks for filing this report. Do you have a source repository that can be used to replicate the old/new behavior?

**Comment by yannikschaelte** [comment]
I don't have a minimal example, but it appears in https://github.com/ICB-DCM/pyABC and https://github.com/ICB-DCM/pyPESTO.

**Comment by benjaoming** [comment]
This could be a variation of #1452

What happens if you downgrade Sphinx to version 5.3.0?

**Comment by yannikschaelte** [comment]
Hi @benjaoming , thanks for your answer. I can confirm that it works with sphinx 5.3.0, rtd theme 1.2.0.

**Comment by benjaoming** [comment]
@yannikschaelte it will probably also work then with Sphinx 6 if you apply the workaround from https://github.com/readthedocs/sphinx_rtd_theme/issues/1452

**Comment by benjaoming** [comment]
...but it'd be interesting to know _if it doesn't_.

**Comment by yannikschaelte** [comment]
Adding the sphinxcontrib.jquery extension indeed solved the issue with sphinx 6.2.1 , I'll consider this a fix here. Thank you!
